src/client/North/Form/NorthForm.jsx

```diff
@@ -31,6 +31,7 @@ const NorthForm = ({ application, applicationIndex, onChange }) => {
           </BreadcrumbItem>
           <BreadcrumbItem active tag="span">
             {applicationId}
+            <small>{` (${id || ''})`}</small>
           </BreadcrumbItem>
```

Comment:
should the 'id' visible to end users?

Reply:
I thought it's not a bad idea, but we can show only the name. Anyway, in the config file the id will be visible, if we need it for some reason

Reply:
maybe it should be only visible when "hovering" on the name.

Reply:
If I remember correctly, we talked about showing it on the UI to make easier identifying the database file, in case we need the client to send it to us for debug/support.

Reply:
Yes, I implemented based on the previous comments on the related issue. Maybe we can remove the id field from the table, but leave it in the breadcrumb, so the user will be able to copy from there if needed

Reply:
In fact, we have a "status" screen for each application/protocol. I think the uuid would be useful there.

Reply:
Indeed, the applicationId/name should be visible in case we need to access the .db file which will have the id in its filename. I agree that this id is better inside the status page